### PR TITLE
Python script for sending telegram messages

### LINKED ISSUE #30: Make a script to send telegram messages with Python
# Description

Script to send messages on telegram with the help of Python.

## Type of issue

- [x] Feature (New Script)

## Checklist:

- [x] I have read the project guidelines.
- [x] I have checked all the existing projects, before submitting a new project issue.
- [x] I have checked previous issues to avoid duplicates.
- [x] This issue will be meaningful for the project.

### PR DESCRIPTION
**Pull request by ameyanrd**
Signed-off-by: Ameya Deshpande <ameyanrd@outlook.com>

# Description

This change adds a python script for sending telegram messages.
The README file is updated on the use of the script.

Fixes #30 

Replace `issue_no` in the above line, with the issue related to this PR.

## Type of change

Choosing one or more options from the following as per the nature of your Pull request.
- NOTE: *These boxes can be checked using **`[X]`***

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Documentation Update

# Checklist:
Please tick all the boxes that are fulfilled by your Pull Request.

- [ ] I have named my files and folder, according to this project's guidelines.
- [ ] My code follows the style guidelines of this project.
- [x] My Pull Request has a descriptive title. (not a vague title like `Update index.md`)
- [X] I have commented on my code, particularly in hard-to-understand areas.
- [X] I have created a helpful and easy to understand `README.md`, according to the given [`README_TEMPLATE.`](https://github.com/Python-World/Python_and_the_Web/blob/master/README_TEMPLATE.md)
- [x] I have included a requirements.txt file (if external libraries are required.)
- [x] My changes do not produce any warnings.
- [ ] I have added a working sample/screenshot of the script.

### CODE REVIEW

**Comment by AdityaJ7** [comment]
Please add a requirements.txt file which contains the external libraries

**Comment by ameyanrd** [comment]
> Please add a requirements.txt file which contains the external libraries

Done.

**Review by AdityaJ7 — APPROVED** [review]

**Review by tusharnankani — APPROVED** [review]
Looks great! 👍🏻

**Comment by AdityaJ7** [comment]
@all-contributors please add @ameyanrd  for code

File: Scripts/Bots/telegram_bot.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import telebot 
+from telethon.sync import TelegramClient 
+from telethon.tl.types import InputPeerUser, InputPeerChannel 
+from telethon import TelegramClient, sync, events 
+  
+   
+# Enter the API token in 'token'.
+# Enter the API ID and API Hash from
+# the telegram app created.
+api_id = 'API_id'
```
Comment (chavarera) [inline Scripts/Bots/telegram_bot.py:11]: store token , API and other configuration  in config.ini  file.

File: Scripts/Bots/telegram_bot.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  
+# your phone number 
+phone = 'YOUR_PHONE_NUMBER_WTH_COUNTRY_CODE'
+   
+# creating a telegram session and assigning 
+# it to a variable client 
+client = TelegramClient('session', api_id, api_hash) 
+   
+# connecting and building the session 
+client.connect()
```
Comment (chavarera) [inline Scripts/Bots/telegram_bot.py:23]: please make your code more readable by adding functions.

**Comment by AdityaJ7** [comment]
@ameyanrd  please create another pr to make the changes in your code as suggested by @chavarera

**Comment by ameyanrd** [comment]
Yeah okay

**Pull request merged**